projects/admin-core/src/lib/routes.ts

```diff
@@ -4,4 +4,6 @@ export enum RoutesEnum {
   CATALOG_SERVICE_DETAILS = 'node/:nodeId/service/:serviceId',
   CATALOG_LAYER_DETAILS = 'node/:nodeId/service/:serviceId/layer/:layerId',
   ADMIN_HOME = '',
+  USER_ADMIN = 'user-admin',
```

Comment:
Minor remark, but just user is enough I think, since (in production) its under the /admin folder anyway, now you would get /admin/user-admin